src/transformers/models/rtdetrv2/modular_rtdetrv2.py

```diff
@@ -0,0 +1,1031 @@
+from ..rt_detr.modeling_rt_detr import (
+    RTDetrDecoderLayer, RTDetrModelOutput, RTDetrObjectDetectionOutput, RTDetrConvEncoder, RTDetrHybridEncoder,
+    RTDetrEncoderLayer, RTDetrConvEncoder, RTDetrMLPPredictionHead, RTDetrDecoderOutput, RTDetrDecoderLayer, 
+    RTDetrMultiheadAttention, RTDetrMultiscaleDeformableAttention
+)
+from ...modeling_utils import PreTrainedModel
+from ..rt_detr.configuration_rt_detr import RTDetrConfig
+from ...activations import ACT2FN
+import math
+import os
+import warnings
+from dataclasses import dataclass
+from functools import lru_cache, partial, wraps
+from pathlib import Path
+from typing import Dict, List, Optional, Tuple, Union
+
+import torch
+import torch.nn.functional as F
+from torch import Tensor, nn
+from torch.autograd import Function
+from torch.autograd.function import once_differentiable
+
+from ...utils import is_torch_cuda_available
+import torch.nn as nn
+
+class RTDetrv2Config(RTDetrConfig):
+    model_type = "rt_detr_v2"
+
+    def __init__(self, 
+                 decoder_n_levels=3,  # default value
+                 decoder_offset_scale=0.5,  # default value
+                 **kwargs):
+        # init the base RTDetrConfig class with additional parameters
+        super().__init__(**kwargs)
```

Comment:
```suggestion
    def __init__(self, 
                 decoder_n_levels=3,  # default value
                 decoder_offset_scale=0.5,  # default value
                 **super_kwargs):
        # init the base RTDetrConfig class with additional parameters
        super().__init__(**super_kwargs)
```
If you want to unpack the args of the parent class, use the magic `**super_kwargs` instead of `**kwargs` here